package.json

```diff
@@ -55,32 +55,34 @@
     "core-js": "2.5.1",
     "echarts": "^4.0.2",
     "eva-icons": "^1.1.3",
+    "install": "^0.13.0",
     "intl": "1.2.5",
     "ionicons": "2.0.1",
     "leaflet": "1.2.0",
     "nebular-icons": "1.1.0",
-    "ng2-ckeditor": "^1.2.2",
+    "ng2-ckeditor": "^1.2.7",
```

Comment:
Should we update the this package when migrating it to next stable version ?
Curious to know if it wont affect the working ?

Reply:
The current version has incompatible dependency so I upgraded it